packages/toolpad-app/src/components/EditableText.tsx

```diff
@@ -1,42 +1,38 @@
 import * as React from 'react';
-import {
-  TextField,
-  Theme,
-  TypographyVariant,
-  SxProps,
-  InputBaseComponentProps,
-} from '@mui/material';
+import { TextField, Theme, TypographyVariant, SxProps } from '@mui/material';
 
 interface EditableTextProps {
   defaultValue?: string;
+  disabled?: boolean;
   editing?: boolean;
-  isError?: boolean;
   errorText?: string;
-  onKeyUp?: (event: React.KeyboardEvent<HTMLInputElement>) => void;
-  onBlur?: (event: React.FocusEvent<HTMLInputElement>) => void;
-  variant?: TypographyVariant;
+  helperText?: React.ReactNode;
+  isError?: boolean;
```

Comment:
Shall we keep the same interface as MUI the `TextField`? just a `error` and `helperText`